aio/content/examples/rx-library/src/simple-creation.2.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { docRegionInterval } from './simple-creation.2';
+
+describe('simple-creation.2', () => {
+  beforeEach(() => {
+    jasmine.clock().install();
+  });
+
+  afterEach(() => {
+    jasmine.clock().uninstall();
+  });
```

Comment:
Let's keep this consistent with other recently added tests:

```suggestion
  beforeEach(() => jasmine.clock().install());
  afterEach(() => jasmine.clock().uninstall());
```